insect:
     # Publish all joint states -----------------------------------

    joint_state_controller:
        type: joint_state_controller/JointStateController
        publish_rate: 50

     # Position Controllers ---------------------------------------
    l_f_hip_joint_position_controller:
        type: effort_controllers/JointPositionController
        joint: l_f_hip_joint
        pid: {p: 5000, i: 0.01, d: 100.0}

    r_f_hip_joint_position_controller:
        type: effort_controllers/JointPositionController
        joint: r_f_hip_joint
        pid: {p: 5000, i: 0.01, d: 100.0}

    l_b_hip_joint_position_controller:
        type: effort_controllers/JointPositionController
        joint: l_b_hip_joint
        pid: {p: 5000, i: 0.01, d: 100.0}

    r_b_hip_joint_position_controller:
        type: effort_controllers/JointPositionController
        joint: r_b_hip_joint
        pid: {p: 5000, i: 0.01, d: 100.0}

    l_f_limb_joint_position_controller:
        type: effort_controllers/JointPositionController
        joint: l_f_limb_joint
        pid: {p: 5000, i: 0.01, d: 100.0}

    r_f_limb_joint_position_controller:
        type: effort_controllers/JointPositionController
        joint: r_f_limb_joint
        pid: {p: 5000, i: 0.01, d: 100.0}

    l_b_limb_joint_position_controller:
        type: effort_controllers/JointPositionController
        joint: l_b_limb_joint
        pid: {p: 5000, i: 0.01, d: 100.0}

    r_b_limb_joint_position_controller:
        type: effort_controllers/JointPositionController
        joint: r_b_limb_joint
        pid: {p: 5000, i: 0.01, d: 100.0}

    #gazebo_ros_control/pid_gains:
      #l_f_hip_joint: {p: 5000.0, i: 0.0, d: 100.0}
      #r_f_hip_joint: {p: 5000.0, i: 0.0, d: 100.0}
      #l_b_hip_joint: {p: 5000.0, i: 0.0, d: 100.0}
      #r_b_hip_joint: {p: 5000.0, i: 0.0, d: 100.0}
      #l_f_limb_joint: {p: 5000.0, i: 0.0, d: 100.0}
      #r_f_limb_joint: {p: 5000.0, i: 0.0, d: 100.0}
      #l_b_limb_joint: {p: 5000.0, i: 0.0, d: 100.0}
      #r_b_limb_joint: {p: 5000.0, i: 0.0, d: 100.0}
